.gitignore

```diff
@@ -1,7 +1,19 @@
 *.pyc
+*.egg
```

Comment:
These changes shouldn't be in the PR, they should be in your global `.gitignore` file in `~/.gitignore`.  We don't want to pollute the repository's gitignore file with things that are specific to your development environment.